Dockerfile

```diff
@@ -1,6 +1,6 @@
 # --- build image
 
-FROM rust:1.80 AS builder
+FROM docker.io/rust:latest AS builder
```

Comment:
Can we still keep pinning to an explicit Rust version? I don't mind bumping to 1.82 but I mind surprises :-)

Reply:
Dropped the Dockerfile changes, as I currently use an alpine based approach:

```Dockerfile
# --- build image

FROM docker.io/rust:alpine AS builder

RUN apk add musl-dev

ENV USER=app
ENV UID=10001

RUN adduser \
    --disabled-password \
    --gecos "" \
    --home "/nonexistent" \
    --shell "/sbin/nologin" \
    --no-create-home \
    --uid "${UID}" \
    "${USER}"

WORKDIR /app
COPY . .
RUN cargo install --path .


# --- final image

FROM scratch

COPY --from=builder /etc/passwd /etc/passwd
COPY --from=builder /etc/group /etc/group

WORKDIR /app
COPY --from=builder /usr/local/cargo/bin/wastebin /app/wastebin
USER app:app
CMD ["/app/wastebin"]
```